storage/object/how-to/get-started-edge-services.mdx

```diff
@@ -0,0 +1,293 @@
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a custom domain for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Endpoint** panel, click **Reset**. 
+
+    A screen displays warning you that this will reset the bucket's domain back to the default Edge Services endpoint. Edge Services will consider your customized subdomain as unknown. You should also remember to delete your CNAME record from your domain provider, and your SSL/TLS certificate.
+
+5. Click **Reset domain** to finish.
+
+## How to configure your cache
+
+The cache feature allows you to cache your bucket's content with Edge Services. This means that content can be served directly to users from Edge Services' servers, enhancing performance. 
+
+You can disable and enable caching at will, as well as control the lifetime of an object in the cache. You can also purge your entire cache, or specific objects within it. A log is displayed to help you track your purge events.
+
+### How to enable your cache
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to configure a cache for.
+
+3. Click the **Edge Services** tab.
+
+    TODO SCREENSHOT
+
+4. In the **Cache** panel, use the <Icon name="toggle" /> icon to enable the cache.
+
+    The **Life time** configuration box displays. This enables you to define, in seconds, how long an object can be stored in the cache before it must be retrieved freshly from the bucket.
+
+    <Message type="tip">
+    As an example, a value of 0 means that objects will not be cached, unless they have a separately-defined caching directive.
+    </Message>
+
+5. Leave the default value of 3600s (1 hour) in place, or enter another value.
+
+The cache is now enabled.
+
+### How to purge all objects from your cache
+
+This clears all objects from your cache. Afterwards, fresh copies must be retrieved from the bucket before Edge Services can store them again in the cache. 
+
+1. Click **Object Storage** in the **Storage** section of the Scaleway console side menu. The list of your buckets displays.
+
+2. Click the name of the bucket you want to purge the cache for.
+
+3. Click the **Edge Services** tab.
+
+4. In the **Purge cache** panel, click **Purge all**. 
+
+        A screen displays warning you that your cache will be emptied, and Edge Services will have to retrieve objects from your bucket before re-caching them. 
+        
+5. Click **Purge cache** to confirm.
+
+### How to purge specific objects from your cache
+
+This allows you to specify the precise objects that you want to clear from the cache. Afterwards, fresh copies must be retrieved from the bucket before Edge Services can store them again in the cache. 
```

Comment:
@RoRoJ 

> fresh copies must be retrieved from the bucket

I fear that saying "must" implies that the user has to do something. This is a completely automated process carried by the edge services node whenever a request for an object that's not in the cache comes in.